folly/experimental/symbolizer/Elf.h

```diff
@@ -35,12 +35,26 @@
 namespace folly {
 namespace symbolizer {
 
+#if !defined(__FreeBSD__)
```

Comment:
This way of organizing the conditions is awkward and only works with a single if/else choice. What if we need to support another OS?

```
#if defined(__FreeBSD__)
...
#else
...
#endif
```